Homeworks/homework_08.1.py

```diff
@@ -0,0 +1,33 @@
+"""8.1. Відловлюємо некоректні символи в списку чисел."""
+
+"""Створіть масив зі строками, які будуть складатися з чисел,
+які розділені комою. Наприклад: [”1,2,3,4”, ”1,2,3,4,50” ”qwerty1,2,3”]
+Для кожного елементу списку виведіть суму всіх чисел
+(створіть нову функцію для цього).
+Якщо є символи, що не є числами (”qwerty1,2,3” у прикладі),
+вам потрібно зловити вийняток і вивести “Не можу це зробити!”
+Використовуйте блок try/except, щоб уникнути інших символів,
+окрім чисел у списку.
+Для цього прикладу правильний вивід буде:
+10, 60, “Не можу це зробити”
+"""
+
+
+def sum_of_nums(string):
```

Comment:
Better to avoid "string" naming for variables - cause present  module with such name 
`import string`